### PR TITLE
add tolerance for tpu in r2 and canberra tests

### PR DESCRIPTION
**Pull request by sdesrozis**
Fixes #1318 #1314 

Description:
Add tolerance for test of metrics r2 and canberra for TPU

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

File: tests/ignite/contrib/metrics/regression/test_canberra_metric.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def test_compute():
     assert canberra.pairwise([v1, v2])[0][1] == pytest.approx(np_sum)
 
 
-def _test_distrib_compute(device):
+def _test_distrib_compute(device, tol=1e-4):
```
Comment (vfdev-5) [inline tests/ignite/contrib/metrics/regression/test_canberra_metric.py:66]: Why 1e-4 by default ? Let's put it a bit stricter 1e-8

Comment (sdesrozis) [reply tests/ignite/contrib/metrics/regression/test_canberra_metric.py:66]: Ok. It was the value in ssim metric, I did copy/paste. So 1.e-8 and 1.e-3 when tpu ?

Comment (vfdev-5) [reply tests/ignite/contrib/metrics/regression/test_canberra_metric.py:66]: Let me run it on TPUs

Comment (vfdev-5) [reply tests/ignite/contrib/metrics/regression/test_canberra_metric.py:66]: Looks like canberra tests pass without any modifications. The problem is with R2 only

**Review by vfdev-5 — APPROVED** [review]
LGTM, @sdesrozis thanks !

**Pull request merged**